paasta_tools/bounce_lib.py

```diff
@@ -375,6 +378,7 @@ def downthenup_bounce(
     happy_new_tasks,
     old_app_live_happy_tasks,
     old_app_live_unhappy_tasks,
+    margin_factor=1,
```

Comment:
set the default to 1.0, just to make it obvious that it's intended to be a float